src/test/kotlin/com/natpryce/hamkrest/describe_tests.kt

```diff
@@ -45,10 +46,13 @@ class DelimitingValuesInStrings {
 
         assert.that(describe(d), equalTo("d"))
     }
-    
+
     @Test
     fun file_paths() {
         val path = Paths.get("/foo/bar/baz")
-        assertThat(describe(path), equalTo("/foo/bar/baz"))
+        assertThat(describe(path), equalTo(
+                File.separator + "foo"
```

Comment:
Good catch.  But now I've looked into the Path API, I find it's more complicated than that.  The test should use `path.fileSystem.separator`, not `File.separator`, because a Path object might refer to a file on a file system other than the system's default

Reply:
Good point, I didn't think that the target platform might be different from the host platform.